portfolio/src/main/webapp/script.js

```diff
@@ -33,3 +33,29 @@ function getGreetingUsingArrowFunctions() {
     document.getElementById('greeting-container').innerText = greeting;
   });
 }
+
+// week2-step3 add json
+function getServerStats() {
+  console.log("Jill~~~ <3");
+  fetch('/data').then(response => response.json()).then((stats) => {
+    // stats is an object, not a string, so we have to
+    // reference its fields to create HTML content
+    console.log(stats);
+
+    const statsListElement = document.getElementById('greeting-container');
+    statsListElement.innerHTML = '';
+    statsListElement.appendChild(
+        createListElement(stats[0]));
```

Comment:
nit: usually we would prefer to have this in one line, unless it is longer a particular line length. 
statsListElement.appendChild(createListElement(stats[0]));

This is just about coding style and people working on same project would prefer to work based on the same coding style guideline.